### ISSUE TITLE
implement softmax layer

### ISSUE THREAD

**Issue by milhidaka** [mil-tokyo/webdnn#254]
Keras converterでsoftmaxは無条件で無視しているが、あまり良い扱いではないので実装しておく

**Comment by milhidaka** [comment]
Released in v1.1.0
